### PR TITLE
Improving Keyboard Shortcuts for the toolbox items

### PR DESCRIPTION
**Pull request by DhairyaBahl**
@ice0 This is the PR for the issue #1794 and in this i almost fixed every keyboard shortcut which i could with inkscape except few which i couldn't find or couldn't understand like mirror vertical and mirror horizontal. they must be named as something else. Kindly tell me that and I will fix that also. or you can check these changes first and fully merge them into code base  then we will move to the ones which were left. Your Call :)

Kindly review this PR . Thank You

### CODE REVIEW

File: synfig-studio/src/gui/app.cpp
Comment (Keyikedalube) [inline synfig-studio/src/gui/app.cpp:1319]: state-fill is bucket 👉🏽 `u`

**Review by Keyikedalube — CHANGES_REQUESTED** [review]
Change the fill accel key

**Comment by Keyikedalube** [comment]
Tools to be discussed:
- Mirror
- Draw
- Cutout
- Width
- Sketch
- Scale
- Rotate
- Polygon

**Comment by DhairyaBahl** [comment]
@Keyikedalube I changed the fill accel key but what should we do about those To Be Discussed Items that you mentioned above. 

Kindly tell me so that I can work further if required. Thanks :)

**Comment by Keyikedalube** [comment]
> what should we do about those To Be Discussed Items

Well, those tool items still need to be changed :) Look at the draft document again and try to come up with new accel keys for those items. Some were already proposed. Since this PR already implements keyboard shortcuts behavior similar to Inkscape, you can fire up Inkscape on your computer and tinker around with it tools (left sidebar) just to get some idea 😄

**Comment by DhairyaBahl** [comment]
@Keyikedalube Okay I am on it :) 

Thanks :)

**Comment by DhairyaBahl** [comment]
@Keyikedalube I was working with inkscape and
1 we have to correct the documentation I guess because d key starts the drawing tool. 
2.i think rest of the shortcuts that are proposed in the documentation are usable and we should go with them.
although i you guys want, I can try to make my own proposal of changes, no issues in that. 
3.as i can see in inkscape, polygons and stars both have same shortcut may be ! its written stars and polygons
4. Kindly tell me what should i do next. apply those keys from the documentation,or propose my own. 

Thanks :)

**Comment by Keyikedalube** [comment]
> we have to correct the documentation

Don't worry about documentation because once this PR is done, that page has to be deleted anyway.

> p key starts the drawing tool

Thanks for noticing that :) Synfig draw tool can use `p`

> i think rest of the shortcuts that are proposed in the documentation are usable and we should go with them

Yes, but try to reserve non-letter characters for canvas operation OK. Because now we use period `.` and comma `,` for frame seeking and then there's bracket left `[` for keyframe prev and bracket right `]` for next keyframe. See? non-letter characters are predominantly used for canvas operation already.
Try to come up shortcut key `a-z` for those remaining **to be discussed** lists

**Comment by Keyikedalube** [comment]
Here's my proposal for:
- scale `l` *<-- that's small L*
- rotate `a`
- mirror `i`

Proposal for keys not mentioned in the doc:
- Zoom `z`
- Skeleton `n`

The rest can use the keys already proposed.

**Comment by DhairyaBahl** [comment]
@Keyikedalube Kindly review the changes. I was also thinking about alternatives for those non-keyboard keys but, according do me , for frame seeking comma and period are best because of "<" and ">" in the key which is relatable with backward and forward. 

Thanks :)

**Review by Keyikedalube — APPROVED** [review]

**Comment by Keyikedalube** [comment]
@DhairyaBahl Great job and thanks for taking up the issue and making a PR :)

Now it's up to maintainers to have the final decision.

**Comment by DhairyaBahl** [comment]
@ice0 @morevnaproject  Kindly review this PR. And suggest if I have to make any changes. Thank you :)

**Comment by ice0** [comment]
Merged. Thank you!

**Pull request merged**
